relayer/pkg/starknet/utils.go

```diff
@@ -74,12 +74,12 @@ func DecodeFelts(felts []*big.Int) ([]byte, error) {
 	length := int(felts[0].Int64())
 
 	for _, felt := range felts[1:] {
-		buf := buf[:Min(chunkSize, length)]
+		bytes := buf[:Min(chunkSize, length)]
 
-		felt.FillBytes(buf)
-		data = append(data, buf...)
+		felt.FillBytes(bytes)
+		data = append(data, bytes...)
 
-		length -= len(buf)
+		length -= len(bytes)
```

Comment:
wouldn't naming the slice `bytes` here conflict with the `bytes` module that is also used?

Reply:
It's just scoped to that small block, can use a different name though, what should we name it?

Reply:
maybe something simple like `byteBuffer` or `buffer` ?